native/acter/src/api/news.rs

```diff
@@ -381,12 +401,37 @@ impl NewsEntry {
             .await?
     }
 
-    pub fn comments_count(&self) -> u32 {
-        4
+    pub async fn comments_count(&self) -> Result<u32> {
+        let me = self.clone();
+
+        RUNTIME
+            .spawn(async move {
+                let manager = me.comments().await?;
+                Ok(manager.stats().total_comments_count().to_owned())
+            })
+            .await?
+    }
```

Comment:
if this triggers the build of `me.comments()` (which is then used on the UI afterwards anyways), the this shouldn't be here but instead only be a helper on the comments_manager ... this avoids us creating that internal thing and loading from the database twice ...

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1208/commits/e5e7c6a86780009aeeba43cbbfa5d677891a4fd9